src/attr.rs

```diff
@@ -292,10 +292,10 @@ impl<'s> Iterator for AttributesIter<'s> {
     }
 }
 
-impl<'s> IntoIterator for &'s Attributes<'s> {
+impl<'s, 'a: 's> IntoIterator for &'a Attributes<'s> {
     type Item = (&'s str, &'s AttributeValue<'s>);
```

Comment:
Similarly here, we want to borrow the values only for the lifetime of the iterator `'a`: `(&'s str, &'a AttributeValue<'s>)`